### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by Wang-zipeng** [foolwood/pytorch-slimming#16]
Question about loss, I have a question about the loss function, in this repo, loss function is only a cross_entropy but in the paper this cross_entropy  and Xγ∈Γ g(γ) , how to understand the different between implementation and the paper ?
